### DISCUSSION TITLE
Searching for Deployment configuration examples

### DISCUSSION THREAD

**Discussion by 64J0** [kubernetes-client/csharp#1520] [Q&A]
Hello, hope you're good. I'm looking for examples of Deployment configuration. Don't know why, but when I try to find some in the repo it does not appear.

**Comment by tg123** [comment]
could you please elaborate on `Deployment configuration` 
are you looking for cm?

**Reply by 64J0** [reply]
> could you please elaborate on Deployment configuration

Sure, I'm looking for representing this kind of manifest https://kubernetes.io/docs/concepts/workloads/controllers/deployment/.

> are you looking for cm?

For cm I understand ConfigMap. This is not what I'm looking for now.
